src/main/java/com/todoist/pojo/Due.java

```diff
@@ -0,0 +1,62 @@
+package com.todoist.pojo;
+
+public class Due {
+    private String date;
+    private String timezone;
+    private String string;
+    private String lang;
+    private boolean isRecurring;
```

Comment:
Given there are no setters, I wonder if we should make all of these final.

Reply:
Makes sense! Fixed in  c4095599524aa1183a1c1cb9dda2926b043720d8